CUSTOMISING.md

```diff
@@ -111,6 +111,26 @@ damap:
       db-kind: postgresql # your database type
 ```
 
+If you want to customize liquibase for your own institutional backend:
+- Create a new folder in db with your institutions name - like e.g. tuwien
+- Create your own root changelog and change the liquibase path in application.yaml to point to your new root changelog
+- Your new root changelog should look something like this:
```

Comment:
i would also show an exmple of what to insert into the application yaml:
```suggestion
Here an example of how to override the root changelog in the application.yaml file of your application:

quarkus:
  liquibase:
    migrate-at-start: true
    change-log: db/tuwien/mainChangeLog.yaml

- Your new root changelog should look something like this:
```
also not forget to make the sample code formatted to yaml

Reply:
Good idea :)